### ISSUE TITLE
[A11y] Explore Collections - Grid - Contrast

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1877]
### Operating System

Windows

### Browser

Edge

### Affected page

Explore Collections

### Admin user?

Yes

### Issue Description

Contrast of white on orange text is failing accessibility checks.

### Suggested Fix

Pick a darker orange?  Or not use orange at all for backgrounds.  Even though black on orange doesn't technically fail WCAG, it's not a great combination.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

A tool that auto-detects contrast can help here, but some flag text incorrectly (WAVE) while others miss some of the contrast failures (Accessibility Insights).  Most likely this will require a manual check with a color-picker.

**Comment by CGillen** [comment]
Fixed by #1987
